tuareg-opam.el

```diff
@@ -224,12 +224,12 @@ See `prettify-symbols-alist' for more information.")
 (require 'flymake)
 
 (defalias 'tuareg-opam--flymake-proc-init-create-temp-buffer-copy
-  (if (functionp #'flymake-proc-init-create-temp-buffer-copy)
```

Comment:
What is the problem here? `functionp` exists since Emacs 20.

Reply:
The code generated a warning about `flymake-proc-init-create-temp-buffer-copy` being undefined in older Emacs versions. `fboundp` is the correct predicate to use for this kind of thing, because it has some extra compiler chumminess that suppresses warnings in the not-taken branch.

Reply:
Addendum: while the above is true, the actual warning in this case came from the use of `#'` which makes the compiler complain if the quoted function doesn't actually exist. While it would technically be possible to write something like `(if (functionp 'some-function) ...)`, it's not really good style: checking whether a particular symbol names a function is what `fboundp` is for.